components/pipedrive/pipedrive.app.mjs

```diff
@@ -1,11 +1,225 @@
+import pipedrive from "pipedrive";
+import constants from "./common/constants.mjs";
+
 export default {
   type: "app",
   app: "pipedrive",
-  propDefinitions: {},
+  propDefinitions: {
+    userId: {
+      type: "integer",
+      label: "User ID",
+      description: "ID of the user who will be marked as the owner of this deal. If omitted, the authorized user ID will be used.",
+      optional: true,
+    },
+    personId: {
+      type: "integer",
+      label: "Person ID",
+      description: "ID of the person this deal will be associated with",
+      optional: true,
+    },
+    organizationId: {
+      type: "integer",
+      label: "Organization ID",
+      description: "ID of the organization this deal will be associated with",
+      optional: true,
+    },
+    probability: {
+      type: "integer",
+      label: "Probability",
+      description: "Deal success probability percentage. Used/shown only when deal_probability for the pipeline of the deal is enabled.",
+      optional: true,
+    },
+    lostReason: {
+      type: "string",
+      label: "Lost Reason",
+      description: "Optional message about why the deal was lost (to be used when status=lost)",
+      optional: true,
+    },
+    visibleTo: {
+      type: "string",
+      label: "Visible To",
+      description: "Visibility of the deal. If omitted, visibility will be set to the default visibility setting of this item type for the authorized user.\n1 - Owner & followers (private)\n3 - Entire company (shared)",
+      optional: true,
+      options: constants.VISIBLE_TO_OPTIONS,
+    },
+    addTime: {
+      type: "string",
+      label: "Add Time",
+      description: "Optional creation date & time of the deal in UTC. Requires admin user API token. Format: YYYY-MM-DD HH:MM:SS",
+      optional: true,
+    },
+    ownerId: {
+      type: "integer",
+      label: "Owner ID",
+      description: "ID of the user who will be marked as the owner of this organization. When omitted, the authorized user ID will be used.",
+      optional: true,
+    },
```

Comment:
If `ownerId` will use the same async options as `userId`, could it just be removed?